src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -9,17 +9,10 @@ def link_to_all
     end
   end
 
-  def filter_by_type_link(link_text, filter_item)
-    link_to(link_text, my_notifications_path(filter_item), class: filter_css(filter_item))
-  end
-
-  def filter_by_project_link(link_text, amount, filter_item)
-    badge_color = notification_filter_active?(filter_item) ? 'badge-light' : 'badge-primary'
+  def filter_notification_link(link_text, amount, filter_item)
     link_to(my_notifications_path(filter_item), class: filter_css(filter_item)) do
-      capture do
```

Comment:
Not sure why we use the `capture` block here. Maybe there is something I'm not aware of, but from my point of view this is not necessary

Reply:
I don't see any Erb code being rendered inside the capture block so I also think this is not really needed.